packages/arb-token-bridge-ui/tests/support/commands.ts

```diff
@@ -337,6 +343,27 @@ export function findSelectTokenButton(
     .should('have.text', text)
 }
 
+export function findTransactionInTransactionHistory({
+  symbol,
+  amount,
+  duration
+}: {
+  symbol: string
+  amount: number
+  duration?: string
+}) {
+  const rowId = `(claimable|deposit)-row-[0-9xabcdef]*-${amount}${symbol}`
```

Comment:
We should initialize it with `rowIdRegexp = new RegExp(...)` for better readability.

Reply:
Make sense